runelite-client/src/main/java/net/runelite/client/ui/ClientUI.java

```diff
@@ -125,7 +125,10 @@
 	private final EventBus eventBus;
 	private final KeyManager keyManager;
 	private Applet client;
+
+	@Getter
 	private ContainableFrame frame;
```

Comment:
Don't do this. Rather, create method showConfirmDialog in ClientUI that will send internal frame